overlord/ifacestate/helpers.go

```diff
@@ -378,6 +378,32 @@ func (m *InterfaceManager) removeSnapSecurity(task *state.Task, instanceName str
 	return nil
 }
 
+func storeHotplugSlot(st *state.State, repo *interfaces.Repository, stateSlots map[string]*HotplugSlotInfo, iface interfaces.Interface, slot *snap.SlotInfo) error {
```

Comment:
we don't use "store" the verb a lot (because it's easy to confuse with "store" name),   I would s/store/add/ here

Reply:
Right, will rename, thanks.